tests/cases/compiler/exportDefaultEnumTargetES2015ModuleAMD.ts

```diff
@@ -0,0 +1,62 @@
+// https://github.com/Microsoft/TypeScript/issues/3792
+// @target: es2015
+// @module: amd

```

Comment:
Our compiler runner supports test variations for the same source. Instead of multiple `nnnTargetXModuleY.ts` files you can do this instead:
```suggestion
// @module: amd, commonjs, system, es2015
```